draft-jholland-quic-multicast.md

```diff
@@ -225,7 +225,15 @@ Note that the hash is on the unencrypted packet because it checks against a spec
 
 # Packet Scheduling
 
-# Stateless Reset
+
+# Connection Termination
+Termination of the unicast connection behaves as described in Section 10 of {{RFC9000}}, with the following notable differences:
+
+* On the client side, termination of the unicast connection means that it MUST leave all multicast channels and discard any state associated with them. Servers MAY stop sending to multicast channels if there are no unicast connections left that are associated with them.
+
+* For determining if a connection is lively, the client MUST only consider packets received on the unicast connection. Any packets received on a multicast channel MUST NOT be used to reset a timer checking if a potentially specified max_idle_timeout has been reached. If the unicast connection becomes idle and the server does not respond to a liveliness check, the client MUST terminate the connection as described above.
```

Comment:
nit: We should probably use the same "liveness" wording from [RFC 9000 10.1](https://www.rfc-editor.org/rfc/rfc9000#name-liveness-testing) instead of "lively" and "liveliness check".